### DISCUSSION TITLE
Intended support for VSCode's new Light/Dark 'Modern' themes?

### DISCUSSION THREAD

**Discussion by paulalexwilson** [eclipse-theia/theia#12723] [General] [answered]
A few months back, VSCode began shipping with new 'modern' themes, which I very much like:

https://code.visualstudio.com/updates/v1_78#_new-default-color-themes

They don't seem to be available currently from what I can see, however I _can_ see the VSCode Dark+ and Light+ themes which now feel old/outdated at least to my eye. Is there an intention to offer the modern themes within Theia?

**Comment by vince-fugnitto** [comment]
@paulalexwilson I believe they already are available?

![image](https://github.com/eclipse-theia/theia/assets/40359487/f3017723-b14f-429a-811d-230e6792d8c4)

They are contributed by https://open-vsx.org/extension/vscode/theme-defaults.

**Reply by paulalexwilson** [reply]
Looks like the last activity on the `eclipse-theia/vscode-builtin-extensions` [extension repository](https://github.com/eclipse-theia/vscode-builtin-extensions/commit/1e8e1b0fc0dc4ae0cac274f94df5abc82b96f0a9#diff-c600edf6ce0739a94a591d68b4a42d84b76a117e0395a1bb88c36aa5ae9024d7) was to update the baseline to the `1.77` [release](https://github.com/microsoft/vscode/tree/1.77.0), however the new themes were released in the 1.78 [release](https://code.visualstudio.com/updates/v1_78#_new-default-color-themes).

**Reply by vince-fugnitto** [reply, marked as answer]
@paulalexwilson the screenshot I shared shows the same theme. It changed name in `1.78.0` after it was finalized.
- https://github.com/microsoft/vscode/blame/1.78.0/extensions/theme-defaults/themes/dark_plus_experimental.json#L3

**Reply by paulalexwilson** [reply]
Ok - so after downloading the VSIX and installing it, and reloading Theia I can see the _Experimental_ version which looks great. Thanks for your help @vince-fugnitto .
